lib/raygun.ts

```diff
@@ -24,8 +25,10 @@ import {
   Tag,
   Transport,
 } from "./types";
+import * as breadcrumbs from "./raygun.breadcrumbs";
+import * as breadcrumbsExpressJs from "./raygun.breadcrumbs.express";
```

Comment:
If there's not many functions in here, we should import it directly instead with *